docs/api/ctx.md

```diff
@@ -1000,6 +1000,31 @@ app.Get("/admin", func(c fiber.Ctx) error {
 })
 ```
 
+An alternative version of the Locals method that takes advantage of Go's generics feature is also available. This version 
+allows for the manipulation and retrieval of local values within a request's context with a more specific data type.
+
+```go title="Signature"
+func Locals[V any](c Ctx, key any, value ...any) V
+```
+
+```go title="Example"
+app.Use(func(c Ctx) error {
+    fiber.Locals[string](c, "john", "doe")
+    fiber.Locals[int](c, "age", 18)
+    fiber.Locals[bool](c, "isHuman", true)
+    return c.Next()
+})
+app.Get("/test", func(c Ctx) error {
+	fiber.Locals[string](c, "john")     // "doe"
+    fiber.Locals[int](c, "age")         // 18
+    fiber.Locals[bool](c, "isHuman")    // true
+return nil
```

Comment:
sorry can you fix the indent here? 1018 uses a tab instead of spaces and 1021 isn't indented. thx

Reply:
ok i'll change it